keanu-project/src/main/java/io/improbable/keanu/vertices/intgr/IntegerVertex.java

```diff
@@ -117,6 +136,31 @@ public IntegerVertex slice(int dimension, int index) {
         return new IntegerSliceVertex(this, dimension, index);
     }
 
+
+    public BoolVertex equalTo(IntegerVertex rhs) {
```

Comment:
I know they're mostly pass through, but it would be nice to have tests covering them just in case

Reply:
Done - please take a look at `IntegerVertexTest#doesEqualTo`